config/arch/bpf.in

```diff
@@ -0,0 +1,9 @@
+# BPF specific configuration file
+
+## no-package
+## select ARCH_SUPPORTS_EITHER_ENDIAN
+## select ARCH_DEFAULT_LE
+## select GCC_VERY_NEW
+## select BINUTILS_VERY_NEW
```

Comment:
It'd be good to replace these `VERY_NEW`s with actual versions as soon as possible. Going on previous releases we won't see GCC 13 until April/May. Binutils on the other hand seem to have a fairly regular release cadence.

Reply:
For now BPF target only makes sense for master branches as it is still under development. Once time comes and if fixes are backported to earlier version I will make that change. ;-)